eos-update-server/eos-update-server.c

```diff
@@ -242,7 +242,7 @@ timeout_data_setup_timeout (TimeoutData *data)
 {
   clear_source (&data->timeout_id);
   if (data->timeout_seconds > 0)
-    data->timeout_id = g_timeout_add_seconds (data->timeout_seconds, timeout_cb, data);
+    data->timeout_id = g_timeout_add_seconds ((guint) data->timeout_seconds, timeout_cb, data);
```

Comment:
It seems like `TimeoutData::seconds` is never set to a negative value anywhere - can we just make it `guint`?

Reply:
It comes from the `--timeout` command line option (see `options_init()`), which can legitimately be negative.

Reply:
Ah, got it (-1 means no timeout, which is reasonable).